facefusion/workflows/image_to_image.py

```diff
@@ -0,0 +1,74 @@
+from facefusion import logger, process_manager, state_manager, wording
+from facefusion.audio import create_empty_audio_frame
+from facefusion.content_analyser import analyse_image
+from facefusion.ffmpeg import copy_image, finalize_image
+from facefusion.filesystem import is_image
+from facefusion.processors.core import get_processors_modules
+from facefusion.temp_helper import clear_temp_directory, get_temp_file_path
+from facefusion.time_helper import calculate_end_time
+from facefusion.types import ErrorCode
+from facefusion.vision import detect_image_resolution, pack_resolution, read_static_image, read_static_images, restrict_image_resolution, scale_resolution, write_image
+from facefusion.workflows.core import is_process_stopping, prepare_temp_directory
+
+
+def process_image(start_time : float) -> ErrorCode:
```

Comment:
it is better to have a general method "process" on every workflow - like  a standard API